examples/convolution_test_cuda.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, kernel, adjointkernel=None):
         self.norm = float(cu_ntuples.sum(cu_ntuples.abs(self.kernel.ntuple)))
         super().__init__(self.space, self.space, linear=True)
 
-    def _apply(self, rhs, out):
+    def _call(self, rhs, out):
```

Comment:
I should do that, I guess. The tests run, but I'll go through the examples, too.